bedrock/base/templates/includes/site-footer.html

```diff
@@ -1,60 +1,67 @@
-            </ul>
-
-            {% if fx_footer_links %}
-              {% if current_platform %}
-                {{ firefox_footer_links(channel=current_channel, platform=current_platform) }}
-              {% else %}
-                {{ firefox_footer_links(channel=current_channel) }}
-              {% endif %}
-            {% endif %}
-
-            <ul class="links-legal">
-              <li><a href="{{ url('privacy') }}" data-link-type="footer" data-link-name="Privacy">{{ _('Privacy') }}</a></li>
-              <li class="wrap"><a href="{{ url('privacy.notices.websites') }}#cookies" data-link-type="footer" data-link-name="Cookies">{{ _('Cookies') }}</a></li>
-              <li class="wrap"><a href="{{ url('legal.index') }}" data-link-type="footer" data-link-name="Legal">{{ _('Legal') }}</a></li>
-              <li class="clear"><a href="{{ url('legal.fraud-report') }}" data-link-type="footer" data-link-name="Report Trademark Abuse">{{ _('Report Trademark Abuse') }}</a></li>
-            </ul>
-          </div>
-          <div class="col col-3">
-            <ul class="links-social">
+              <li><a href="{{ url('mozorg.contact.contact-landing') }}" data-link-type="footer" data-link-name="Contact Us">{{ _('Contact Us') }}</a></li>
+              <li><a href="{{ donate_url('footer') }}" class="donate" data-link-type="footer" data-link-name="Donate">{{ _('Donate') }}</a></li>
+              <li><a href="https://wiki.mozilla.org/Webdev/GetInvolved/mozilla.org" data-link-type="footer" data-link-name="Contribute to this site">{{ _('Contribute to this site') }}</a></li>
               <li>
-                {{ _('Mozilla:') }}
-                <ul>
+                {{ _('Social:') }}
+                <ul class="social-links">
                   <li><a href="https://twitter.com/mozilla" data-link-type="footer" data-link-name="Twitter (@mozilla)">{{ _('Twitter') }}<span> (@mozilla)</span></a></li>
                   <li><a href="https://www.facebook.com/mozilla" data-link-type="footer" data-link-name="Facebook (Mozilla)">{{ _('Facebook') }}<span> (Mozilla)</span></a></li>
                   <li><a href="https://www.instagram.com/mozillagram/" data-link-type="footer" data-link-name="Instagram (@mozillagram)">{{ _('Instagram') }}<span> (@mozillagram)</span></a></li>
                 </ul>
               </li>
+            </ul>
+          </div>
+          <div class="col col-3">
+            <strong>{{ _('Firefox') }}</strong>
+            <ul class="firefox-links">
+              <li><a href="{{ url('firefox.new') }}" data-link-type="footer" data-link-name="Download Firefox now">{{ _('Download Firefox now') }}</a></li>
+              <li><a href="{{ url('firefox') }}" data-link-type="footer" data-link-name="Firefox Web Browser">{{ _('Firefox Web Browser') }}</li></a>
```

Comment:
The Firefox content hub is still a far off thing, and may not end up living at /firefox. Until that time, I don't think we should add duplicate links just for SEO. We could always add it later once the page exists and we know the URL.

Please keep the link to /new aimed at scene1. Pointing a regular link at scene2 will not work in older browsers, which is something our dedicated download buttons mitigate for us. I don't think we should add a normal link to scene2 like this from every page on the site.